src/cdk/platform/platform.ts

```diff
@@ -37,6 +50,13 @@ export class Platform {
   isBrowser: boolean = this._platformId ?
       isPlatformBrowser(this._platformId) : typeof document === 'object' && !!document;
 
+  /** Whether the application is running inside a testing environment. */
+  isTest: boolean =
```

Comment:
I don't know that I want to make this a public API. I worry that it will put us in a hard position in the future with respect to not breaking people. Our own internal usage of this minor enough that I'm much less worries about breaking changes there.

Reply:
We could prefix it with an underscore, or the barrel o character used by Ivy instructions (too avoid the member showing up in autocomplete)

Alternatively we could also just expose an underscore-prefixed method from cdk/platform

Reply:
I would probably lean towards an underscore-prefixed function in the cdk/platform package